### ISSUE TITLE
Windows issue with 1.7.0 release

### ISSUE THREAD

**Issue by nezix** [ospray/ospray#273]
I tested 3 different Windows 10 installations of OSPRay 1.7.0 and I get the same issue trying to start ospExampleViewer:

```
#osp: INITIALIZATION ERROR --> could not open module lib ospray_module_ispc: The specified module could not be found.

#osp: INITIALIZATION ERROR --> Could not find device of type: default.  Make sure you have the correct OSPRay libraries linked.
FATAL ERROR DURING INITIALIZATION!
```

The ospray_module_ispc.dll and .lib are in the same folder.

I also compiled the current master branch and the devel branch with Visual Studio 2017 but got the same issue again.

**Comment by jeffamstutz** [comment]
Did this issue show up in ```v1.6.1``` or earlier? And are you using the zip or installer version of the binary release?

**Comment by nezix** [comment]
It started with 1.7.0. I tried the installer and a cloned version (I guess it is the same as the zip file).
It used to work fine with the 1.6.1.

**Comment by nezix** [comment]
I tested 1.6.1 and it works fine with the installer.
I tried the zip file and the installer for 1.7 and I keep having the same error when I start one of the examples.

**Comment by jeffamstutz** [comment]
I finally got around to replicating this on a Windows machine. As it turns out, it's actually an Embree issue, where the packaged Embree was linking to ```libmmd.dll``` unintentionally. This is resolved by using Embree v3.2.2 or later.

Embree issue can be found here: https://github.com/embree/embree/issues/213

I'm prepping a v1.7.1 release at the moment, where I'm using the latest Embree in the packages.

**Comment by jeffamstutz** [comment]
Thus a fix for doing a manual install is to use the latest Embree (v3.2.3).

**Comment by jeffamstutz** [comment]
Upgrading an Embree binary by substituting a .dll should work, but I am not sure of all the consequences of doing this within ParaView. We had a binary incompatibility issue with Embree v3.6.0 (which we thought got fixed by using `v3.6.1`), but none of us have tried this without recompiling. Could you try instead using `v3.2.2` (or `v3.5.2`)?

You may have to upgrade to a newer ParaView (`v5.7.0`): Kitware builds everything from source with their own compilation/packaging environment, so we don't directly deliver any binaries to them (it's semi-out of our control).
